iaso/migrations/0324_auto_20250514_1244.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.20 on 2025-05-14 12:44
+
+from django.db import migrations
+
+
+def create_feature_flags(apps, schema_editor):
+    FeatureFlag = apps.get_model("iaso", "FeatureFlag")
+    FeatureFlag.objects.get_or_create(
+        code="MOBILE_USE_ETHIOPIC_CALENDAR",
+        name="Mobile: Use Ethiopic Calendar.",
+    )
+    FeatureFlag.objects.get_or_create(
+        code="MOBILE_SELECT_CLOSEST_ORG_UNIT",
+        name="Mobile: Propose to jump to closest OrgUnit.",
+    )
+    FeatureFlag.objects.get_or_create(
+        code="MOBILE_SELECT_CLOSEST_ORG_UNIT",
+        name="Mobile: Propose to jump to closest OrgUnit.",
+    )
+    FeatureFlag.objects.get_or_create(
```

Comment:
This seems duplicated:

```suggestion
    FeatureFlag.objects.get_or_create(
        code="MOBILE_SELECT_CLOSEST_ORG_UNIT",
        name="Mobile: Propose to jump to closest OrgUnit.",
    )
```